public/app/features/explore/state/query.ts

```diff
@@ -221,9 +221,19 @@ export function addQueryRow(exploreId: ExploreId, index: number): ThunkResult<vo
  * Cancel running queries
  */
 export function cancelQueries(exploreId: ExploreId): ThunkResult<void> {
-  return (dispatch) => {
+  return (dispatch, getState) => {
     dispatch(scanStopAction({ exploreId }));
     dispatch(cancelQueriesAction({ exploreId }));
+    dispatch(
```

Comment:
Should this be guarded by the feature flag? Or is it just noop in that case?

Reply:
The toggle is now used only to switch it off for Loki (though the name of the flag didn't change). This will be a no-op for Loki when the toggle is not added, but we still need this for Elastic which is not guarded by the toggle.